README.md

```diff
@@ -1,5 +1,13 @@
 # mo_pack
 
+Support for packing formats used to encode the data payloads of UM "fields".
```

Comment:
It'd be nice to get the word "Python" in there somewhere. Perhaps "A Python package providing support for the ..."